components/ModalDelete.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+
+export default function ModalDelete({ onConfirm, onCancel, id }) {
```

Comment:
```suggestion
export default function ModalDelete({ onConfirm, onCancel }) {
```
You could pass down the id, but right now `handleDeleteConfirm` in `Workout/index.js` isn't designed to make use of it

Reply:
For file structure consistency, placing `ModalDelete` in _components/ModalDelete/index.js_ would be more fitting